app/views/scaptimony_policies/index.html.erb

```diff
@@ -2,7 +2,7 @@
 <% title _("Compliance Policies") %>
 
 <% title_actions(
-  display_link_if_authorized(_("New Compliance Policy"), hash_for_new_scaptimony_policy_path),
+   display_link_if_authorized(_("New Compliance Policy"), hash_for_new_scaptimony_policy_path),
```

Comment:
It was actually auto indented by rubymine (which I'm gonna stop
using!) :smiling_imp:

2015-06-11 15:21 GMT+03:00 Marek Hulán <notifications@github.com>:

> In app/views/scaptimony_policies/index.html.erb
> <https://github.com/theforeman/foreman_openscap/pull/109#discussion_r32214522>
> :
>
> > @@ -2,7 +2,7 @@
> >  <% title _("Compliance Policies") %>
> >
> >  <% title_actions(
> > -  display_link_if_authorized(_("New Compliance Policy"), hash_for_new_scaptimony_policy_path),
> > +   display_link_if_authorized(_("New Compliance Policy"), hash_for_new_scaptimony_policy_path),
>
> nit - extra whitespace?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/theforeman/foreman_openscap/pull/109/files#r32214522>.
>